mu/virtual_environment.py

```diff
@@ -376,9 +381,13 @@ def run_subprocess(self, *args, **kwargs):
         logger.debug(
             "Process returned %d; output: %s",
             process.returncode,
-            compact(process.stdout.decode("utf-8")),
+            compact(
+                process.stdout.decode(sys.stdout.encoding, errors="replace")
+            ),

```

Comment:
There are more cases in the crash reports where I *think* some pip stdout/stderr might be missing. I'll start capturing new ones here: https://github.com/mu-editor/mu/issues/1472#issuecomment-839654636

Reply:
Unless I've missed something (certainly possible) I'm not sure what I can do differently here. It might be that my breaking out stdout/err into two streams will help. I don't think it will hinder, so let's go for that and hope it improves things.